app/main.py

```diff
@@ -2,0 +1,32 @@
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> None:
+        def __repr__(self) -> str:
+            return f"{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}"
+
+    def die(self) -> None:
+        if self in Animal.alive:
+            Animal.alive.remove(self)
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = not self.hidden
```

Comment:
You are using the `self` keyword to access the class attribute `hidden`. As per the checklist, you should use the class name to access class attributes. So, it should be `Herbivore.hidden`.